src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -79,17 +79,30 @@ const modalScreenListeners = {
 };
 
 const propTypes = {
+
+    /** Is a workspace currently being created for the user? */
+    isCreatingWorkspace: PropTypes.bool,
+
     /** Information about the network */
     network: PropTypes.shape({
         /** Is the network currently offline or not */
         isOffline: PropTypes.bool,
     }),
 
+    /** Session info for the currently logged in user. */
+    session: PropTypes.shape({
+
+        /** Currently logged in user email */
+        email: PropTypes.string,
+    }),
+
     ...windowDimensionsPropTypes,
 };
 
 const defaultProps = {
+    isCreatingWorkspace: false,
     network: {isOffline: true},
+    session: {email: null},
```

Comment:
Here we use [_.isNull](https://underscorejs.org/#isNull) which will return false if email is undefined, so it could determine that we are logging in as a new user when we are not. I think that's why I did this
https://github.com/Expensify/App/blob/604bd2ec450f28c2bc25814542980113f19f9de1/src/libs/Navigation/AppNavigator/AuthScreens.js#L215